src/bp/admin/ui/src/management/licensing/index.tsx

```diff
@@ -217,6 +224,9 @@ class LicenseStatus extends React.Component {
       return this.renderProDisabled()
     }
 
+    // @ts-ignore
+    const nodes = Number(this.license.limits.nodes)
```

Comment:
Limits is an array of Limit in the typings. This sounds like a really counter intuitive move, don't you think? Maybe we should edit/extend the typings instead?

If you look at line 44, it is possible for `this.license` to be `undefined` which would result in this constant to be set to `NaN`.

Reply:
I know that typings doesn't match, but it was working this way, so maybe the typings are wrong, but I would prefer to address this the next time we play in licensing so we can test it thoroughly

Reply:
Alright! Can you add a TODO then?

Reply:
Good point